src/scenes/UserProfile/UserProfile.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+import NotFound from 'scenes/NotFound'
+import Loading from 'components/Loading'
+import Card, { CardText } from 'components/Card'
+
+import { getAllPersonalData } from './queries'
+
+const UserProfile = props => {
+  if (!props.data) {
+    props.history.push('/auth/login')
+  }
+
+  const { loading, error, User } = props.data
+
+  if (loading) {
+    return <Loading />
+  }
+
+  if (error) {
+    return <div>Error :(</div>
+  }
+
+  if (User === null) {
+    return <NotFound {...props} />
+  }
+
+  const userLog = ['questions', 'answers', 'flags']
+    .map(entityType => User[entityType].map(({id, createdAt, node}) => ({
+      type: {
+        questions: 'Ask',
+        answers: 'Answer',
+        flags: 'Flag'
+      }[entityType],
+      id,
+      at: createdAt,
+      question: {
+        link: `/q/${node.question.slug}-${node.id}`,
+        title: node.question.title
+      }
+    })))
+    .reduce((all, entities) => all.concat(entities))
+    .sort((a, b) => a.at < b.at ? 1 : a.at > b.at ? -1 : 0)
+
+  return (
+    <div>
+      <Card>
+        <CardText>
+          <h1>Profile</h1>
+          <p>
+            This page displays all your personal data (as defined by the GDPR) processed by FAQ.
+            In the future, you will be able to edit, delete and download this data at your convenience.
+          </p>
+          <h1>Identity</h1>
+          <div className="card-form">
+            <label htmlFor="name">Name</label>
+            <input name="name" className="card-input" value={User.name} onChange={() => {}} readOnly />
+            <label htmlFor="givenName">Given name</label>
+            <input name="givenName" className="card-input" value={User.givenName} onChange={() => {}} readOnly />
+            <label htmlFor="familyName">Family name</label>
+            <input name="familyName" className="card-input" value={User.familyName} onChange={() => {}} readOnly />
+            <label htmlFor="email">Email address</label>
+            <input name="email" className="card-input" value={User.email} onChange={() => {}} readOnly />
+            <label htmlFor="picture">Picture link</label>
+            <input name="picture" className="card-input" value={User.picture} onChange={() => {}} readOnly />
+          </div>
+          <h1>Log</h1>
+          <table className="card-table">
+            <thead>
+              <tr>
+                <td>Action</td>
+                <td>When</td>
+                <td>Question</td>
+              </tr>
+            </thead>
+            <tbody>
+              {userLog.map(({type, id, at, question}) => (
+                <tr key={id}>
+                  <td>{type}</td>
+                  <td style={{whiteSpace: 'nowrap'}}>{at}</td>
```

Comment:
You can use the `moment` library to format the timestamp:
moment(at).format('DD/MM/YYYY HH:MM') (See [moment's documentation](https://momentjs.com/docs/#/displaying/format/) for more formats)

Reply:
The format you mention only makes sense in France. I'd rather stick with ISO 8601.